### DISCUSSION TITLE
Option to Disable Star Filter when Searching for Beatmaps

### DISCUSSION THREAD

**Discussion by CrypticEntity** [ppy/osu#15245] [Ideas] [answered]
what it says on the tin. kind of a pain to go to the options to disable the beatmap star filter whenever finding a song that doesnt fit it.

**Comment by gagahpangeran** [comment]
You can use keyword `star` when searching like `star>4` for filtering.

**Comment by peppy** [comment, marked as answer]
I really think the global filter needs to go, and instead be moved local to song select which it is part of the filter control. It's too hidden and too global currently.
